shared/haml/hamburger.haml

```diff
@@ -1,5 +1,7 @@
 :ruby
   options = { level: level, script_level: script_level, language: language, user_type: user_type, loc_prefix: loc_prefix, request: request }
+  user_header_options = { loc_prefix: loc_prefix }
+  dashboard ||= false
```

Comment:
Where does this get set to true?

Reply:
I found a similar use case here in the same directory, but am not entirely sure where it's being set initially: https://github.com/code-dot-org/code-dot-org/blob/87b33e2e91d1dc35df0770245b91fc1fedfc5602/shared/haml/cookie_banner.haml#L17-L18

Reply:
I wonder if it's worth explicitly setting this variable [here](https://github.com/code-dot-org/code-dot-org/blob/058d7bc462436e33ca3de0a435672fe933f05893/dashboard/app/views/layouts/_header.html.haml#L93)

Reply:
Oh nice ok! I can add this to the list of cleanup tasks if/when the experiment is over and we implement a more permanent solution.

Reply:
Sounds good. Thanks!